std/special/bootstrap.zig

```diff
@@ -139,7 +142,7 @@ inline fn callMain() u8 {
             };
             return 0;
```

Comment:
This is a bug.  if main returned a non-zero value, you're ignoring it.

Reply:
Thanks for noticing that! Fixed.